bin/radical-pilot-bridge

```diff
@@ -168,8 +171,13 @@ if __name__ == "__main__":
     reg_addr = sys.argv[2]
     uid      = sys.argv[3]
 
-    ru.daemonize(main=main, args=[sid, reg_addr, uid],
+    evt      = mp.Event()
+    ru.daemonize(main=main, args=[sid, reg_addr, uid, evt],
                  stdout='%s.out' % uid, stderr='%s.err' % uid)
+
+    evt.wait(3.0)
```

Comment:
I added the wait because it was too fast ;-)  I'll increase the timeout to 5 seconds, but would rather like to investigate when that ever takes more than a second, really.